src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -185,26 +172,35 @@ public InternalEngine(EngineConfig engineConfig, boolean skipInitialTranslogReco
         logger.trace("created new InternalEngine");
     }
 
+    private Translog openTranslog(EngineConfig engineConfig, IndexWriter writer, boolean createNew) throws IOException {
+        final Translog.TranslogGeneration generation = loadTranslogIdFromCommit(writer);
+        Translog translog;
+        TranslogConfig translogConfig = engineConfig.getTranslogConfig();
+        if (createNew) {
```

Comment:
no create new will wipe the old one - I think the forceNewTranslog was added by boaz to have a way to drop a translog if something goes horribly wrong. I wonder if we should keep it at all.